File: app/src/main/java/com/lostincontext/commons/images/palette/PaletteImageViewTarget.java
package com.lostincontext.commons.images.palette;


import android.widget.ImageView;

import com.bumptech.glide.request.target.ImageViewTarget;

public class PaletteImageViewTarget extends ImageViewTarget<PaletteBitmap> {


    public PaletteImageViewTarget(ImageView view) {
        super(view);
    }

    @Override protected void setResource(PaletteBitmap resource) {
        view.setImageBitmap(resource.bitmap);
    }
}

File: app/src/main/java/com/lostincontext/commons/images/palette/PaletteImageViewTarget.kt
package com.lostincontext.commons.images.palette


import android.widget.ImageView

import com.bumptech.glide.request.target.ImageViewTarget

class PaletteImageViewTarget(view: ImageView) : ImageViewTarget<PaletteBitmap>(view) {

    override fun setResource(resource: PaletteBitmap) {
        view.setImageBitmap(resource.bitmap)
    }
}
